### ISSUE TITLE
IterableDataset Error

### ISSUE THREAD

**Issue by Lucyhahahahha** [One-paper-luck/MG-Transformer#3]
作者您好，多有叨扰，烦请谅解。请问为什么运行train.py时，会对    dataloader_train = DataLoader(train_dataset, batch_size=args.batch_size, shuffle=True, num_workers=args.workers,drop_last=True)报错呢？
报错内容说ValueError: DataLoader with IterableDataset: expected unspecified shuffle option, but got shuffle=True
已经在网络上搜索很久，仍然没找到解决方案。希望作者指点一二，非常感谢。

**Comment by One-paper-luck** [comment]
没遇到过这个问题。我在重新下载项目，在本地可以跑通，说明代码没有问题。可能是你的环境没装好，建议你装个环境。之前碰到过一些稀奇古怪的问题，大多是环境没配好。

**Comment by One-paper-luck** [comment]
> 作者您好，多有叨扰，烦请谅解。请问为什么运行train.py时，会对 dataloader_train = DataLoader(train_dataset, batch_size=args.batch_size, shuffle=True, num_workers=args.workers,drop_last=True)报错呢？ 报错内容说ValueError: DataLoader with IterableDataset: expected unspecified shuffle option, but got shuffle=True 已经在网络上搜索很久，仍然没找到解决方案。希望作者指点一二，非常感谢。

没遇到过这个问题。我重新下载项目，在本地可以跑通，说明代码没有问题。可能是你的环境没装好，建议你重装个环境。之前碰到过一些稀奇古怪的问题，大多是环境没配好。

**Comment by Lucyhahahahha** [comment]
好的，谢谢您
